precompiles/erc20/integration_test.go

```diff
@@ -83,7 +83,7 @@ func TestIntegrationSuite(t *testing.T) {
 	RunSpecs(t, "ERC20 Extension Suite")
 }
 
-var _ = Describe("ERC20 Extension -", func() {
```

Comment:
@ramacarlucho why was this changed to ordered?

Reply:
I reverted it - there's no need for it with the way that the tests are structured